replications/internal/queue_management.go

```diff
@@ -126,3 +132,106 @@ func (qm *durableQueueManager) UpdateMaxQueueSize(replicationID platform.ID, max
 
 	return nil
 }
+
+// StartReplicationQueues updates the durableQueueManager.replicationQueues map, fully removing any partially deleted
+// queues (present on disk, but not tracked in sqlite), opening all current queues, and logging info for each.
+func (qm *durableQueueManager) StartReplicationQueues(trackedReplications *influxdb.Replications) error {
```

Comment:
Once we land on a signature for it, I think the logic in here is complex enough to warrant some unit tests